spec/dummy/app/views/pages/index.html.erb

```diff
@@ -39,16 +22,50 @@
 
 <% cache @app_props_server_render do %>
   <% puts "=" * 80 %>
-  <% puts "server rendering react component" %>
+  <% puts "server rendering react components" %>
   <% puts "=" * 80 %>
   <!-- Default for prerender is true for the app, set in config/react_on_rails.rb -->
   <%= react_component("App", @app_props_server_render, trace: true) %>
+  <hr/>
+
+  <h1>Server Rendered/Cached React Component Without Redux</h1>
+  <code>
+    <%% cache @app_props_server_render do %><br/>
+    <%%    = react_component("HelloWorld", @app_props_server_render, trace: true) %><br/>
```

Comment:
Maybe it's confusing that we call it HelloWorld in one case, and HelloWorldComponent in another case. Why is this done? Maybe I'm missing something. Be sure to note that ClientApp.jsx is ONLY loaded client side in the browser, and ServerApp.jsx is ONLY loaded server side by the rails server, so there would be no naming conflict.